certora/specs/AccessControlDefaultAdminRules.spec

```diff
@@ -203,6 +269,24 @@ rule beginDefaultAdminTransfer(env e, address newAdmin) {
     "pending default admin delay is set";
 }
 
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Rule: A default admin can't change in less than the applied schedule                                                │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+rule pendingDefaultAdminDelayEnforced(env e1, env e2, method f, calldataarg args, address newAdmin) {
+  require intervalSanity(e1, e2);
+
+  uint48 delayBefore = defaultAdminDelay(e1);
+  address adminBefore = defaultAdmin();
+  beginDefaultAdminTransfer(e1, newAdmin);
+  f(e1, args);
```

Comment:
Did you mean `e2` here? Otherwise `e2` is basically not used in the rule. I think as writen this rule is vacuous, because `adminAfter == newAdmin` will always be false.

Reply:
I share the concern about e2 ... but I don't get the rest of the comment

Reply:
> Did you mean e2 here? 
Yes.